### PR TITLE
fix(Header): inactive section fallback

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- fix header when dont have menu for the url.

### CODE REVIEW

**Comment by cbernat** [comment]
👍

**Comment by cbernat** [comment]
How about adding some tests about this?

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ const Header = ({
   location: { pathname },
 }) => {
   const updatedNav = getUpdateMenu(pathname, nav);
-  const isInactiveSection = !getCurrentPageForUrl(pathname).menu;
+  let isInactiveSection = getCurrentPageForUrl(pathname);
+  isInactiveSection = !isInactiveSection || !isInactiveSection.menu;
```
Comment (lucianaDilizio) [inline src/components/Header/Header.js]: What about:
```
let currentPageforUrl = getCurrentPageForUrl(pathname); 
let isInactiveSection = !currentPageforUrl || !currentPageforUrl .menu;
```
?
I mean, to be clearer.

**Pull request merged**